addons/l10n_kz/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="chart_template" model="account.chart.template">
+        <field name="name">План счетов</field>
+        <field name="code_digits">4</field>
```

Comment:
you sure? 
That field is used to add right-padding with '0' to account codes. 6 is often the minimum

Reply:
They use 4 digits. Check it here https://adilet.zan.kz/rus/docs/V070004771_

![image](https://user-images.githubusercontent.com/186131/145810401-fcb614f2-2a7d-4464-98b2-ed9cf528a321.png)

Reply:
Look at the account template for Belgium, for instance: https://www.plancomptablebelge.be/
There are only 2/3 digits, but 6 digits are used in practice

Reply:
Apparently, it's not the case in post-soviet countries

![image](https://user-images.githubusercontent.com/186131/146526393-682b243f-8ced-4a3a-8787-5ef93814b380.png)

https://www.youtube.com/watch?app=desktop&v=VRmGYHbDLi8